ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,225 @@
+
+            version_config = {
+                'project_name': args.package_name,
+            }
+            create_template_file(
+                'cmake/ConfigVersion.cmake.in.em',
+                package_directory,
+                args.package_name + 'ConfigVersion.cmake.in',
+                version_config)
+
+        if args.build_tool == 'ament_cmake':
+            create_template_file(
+                'ament_cmake/package.xml.em',
+                package_directory,
+                'package.xml',
+                package_xml_config)
+
+            cmakelists_config = {
+                'project_name': args.package_name,
+                'dependencies': args.dependencies,
+                'cpp_node_name': args.cpp_node_name + '.cpp',
+                'cpp_library_name': args.cpp_library_name + '.cpp',
+            }
+            create_template_file(
+                'ament_cmake/CMakeLists.txt.em',
+                package_directory,
+                'CMakeLists.txt',
+                cmakelists_config)
+
+        if args.cpp_node_name or args.cpp_library_name:
+            src_folder = create_folder('src', package_directory)
+            if not src_folder:
+                print('unable to create folder', package_directory)
+                return False
+
+        if args.cpp_node_name:
+            cpp_node_config = {
+                'package_name': args.package_name,
+            }
+            create_template_file(
+                'cpp/main.cpp.em',
+                src_folder,
+                args.cpp_node_name + '.cpp',
+                cpp_node_config)
+
+        if args.cpp_library_name:
+            include_folder = create_folder('include/' + args.package_name, package_directory)
+            class_name = args.cpp_library_name.replace('_', ' ').title()
+            class_name = ''.join(x for x in class_name if not x.isspace())
+            cpp_header_config = {
+                'package_name': args.package_name,
+                'library_name': args.cpp_library_name,
+                'class_name': class_name,
+            }
+            create_template_file(
+                'cpp/header.hpp.em',
+                include_folder,
+                args.cpp_library_name + '.hpp',
+                cpp_header_config)
+
+            cpp_library_config = {
+                'package_name': args.package_name,
+                'library_name': args.cpp_library_name,
+                'class_name': class_name
+            }
+            create_template_file(
+                'cpp/library.cpp.em',
+                src_folder,
+                args.cpp_library_name + '.cpp',
+                cpp_library_config)
+
+            visibility_config = {
+                'package_name': args.package_name.upper(),
+            }
+            create_template_file(
+                'cpp/visibility_control.h.em',
+                include_folder,
+                'visibility_control.h',
+                visibility_config)
+
+        return True
```

Comment:
Related to the previous comment. I think this line needs to be removed (so that the function return `None` in case of success).